src/mame/atlus/cave.h

```diff
@@ -71,6 +71,7 @@ class cave_state : public driver_device
 	void paccarn(machine_config &config);
 	void paceight(machine_config &config);
 	void pacslot(machine_config &config);
+	void jumbogod(machine_config &config);
```

Comment:
The machine_config methods in this section are in alphabetical order, so please move the jumbogod one accordingly